Library/ViewModels/CommentsViewModel.swift

```diff
@@ -139,9 +139,9 @@ public final class CommentsViewModel: CommentsViewModelType,
     let pullToRefresh = self.refreshProperty.signal
 
     let requestFirstPageWith = Signal.merge(
-      initialProject,
-      initialProject.takeWhen(retryFirstPage),
-      initialProject.takeWhen(pullToRefresh)
+      projectOrUpdate,
+      projectOrUpdate.takeWhen(retryFirstPage),
+      projectOrUpdate.takeWhen(pullToRefresh)
```

Comment:
Can we write a test for this when no project is sent in, but only an update and see that comments are returned/ paginated, can pull to refresh?
I believe the tests are already written, it's just a matter of updating the input to be an `Update` instead of a `Project`.

Reply:
There are already some tests check `func testUpdatePagination_WhenLimitReached_CommentsAreLoadedIntoDataSource()` and some others below it.

Reply:
ok, good point, looks good to me!